include/finufft/utils_precindep.h

```diff
@@ -12,10 +12,10 @@
 namespace finufft {
   namespace utils {
   
-  BIGINT next235even(BIGINT n);
+  FINUFFT_EXPORT BIGINT FINUFFT_CDECL next235even(BIGINT n);
 
   // jfm's timer class
-  class CNTime {
+  class FINUFFT_EXPORT CNTime {
```

Comment:
Then can this code be and moved to another header and make a test library that is linked only with tests?
This library has a define `FINUFFT_TESTING` and the remainder of the code has a 
```cpp
#ifdef FINUFFT_TESTING
#include "test_utils.h"
#endif
// the timing function could be wrapped in a macro directly that is defined ad a no op in case  e.g.
#ifdef FINUFFT_TESTING
#define FINUFFT_TIME_START(arg) time_function(arg)
#else
#define FINUFFT_TIME_START(arg) 
#endif
```

Reply:
The more we can move to tests so that it does not get pre-processed, parsed, compiled and linked, the betted.